### ISSUE TITLE
Memory issue with particles (theoratical)

### ISSUE THREAD

**Issue by Jelmer1887** [Pluizebol15/Teamwork-makes-the-dreamwork#3]
**TL:DR: the program keep loading copies of particle-sprites into memory, while not de-loading them when done. MEMORY LEAK**

So atm, when a particle is spawned (the arrow), the game loads it's sprite into memory, and then the draw function draws that sprite.
This is a problem, because, for every new instance of that particle (every new particle appearing on screen), these images are re-loaded into memory, and stored as a new 'image object', all while long gone version of those particles still have their version of that image loaded in memory. This is a memory leak, as we destroy the variables associated with those images when removing them from the entity list.
Every '`Loading ARF_F_#nr...Complete!`' in the image adds a memory leak of roughly a kilobyte...
![image](https://user-images.githubusercontent.com/66646667/89135436-e591be80-d52d-11ea-9491-625a42e6cf18.png)

The solution would be pre-loading the sprites (while we are also loading the other sprites for other things), and referencing to those sprites in every particle instance. I just couldn't 'be bothered yet and it's an easy fix for someone else to try...
